build/utils.js

```diff
@@ -51,8 +60,17 @@ exports.cssLoaders = function (options) {
     css: generateLoaders(),
     postcss: generateLoaders(),
     less: generateLoaders('less'),
-    sass: generateLoaders('sass', { indentedSyntax: true }),
-    scss: generateLoaders('sass'),
+    sass: generateLoaders('sass', {
+      indentedSyntax: true,
+      includePaths: [
+        resolve('node_modules')
+      ]
+    }),
+    scss: generateLoaders('sass', {
+      includePaths: [
+        resolve('node_modules')
+      ]
+    }),
```

Comment:
# Requested Change
Was only necessary for importing files from material-components-web lib. This is no longer being used and should be removed